tests/profile/request/attribute/attribute.spec.js

```diff
@@ -82,10 +79,11 @@ describe('SandboxAttribute', () => {
     const expectedData = {
       name: SOME_NAME,
       value: SOME_VALUE,
-      optional: true,
     };
 
     expect(JSON.stringify(sandboxAttribute))
       .toBe(JSON.stringify(expectedData));
+
+    expect(sandboxAttribute.getOptional()).toBe(true);
```

Comment:
Is this just checking that the property is there, but that we don't serialize it into JSON?

Reply:
Yes, it's just checking we haven't broken `getOptional()`, which is no longer used by `toJSON` - these getters are redundant, but we can't remove them now - we could remove them in the next major.

Reply:
I've reworded the test description on 08877af